bin/kano-sync

```diff
@@ -67,58 +69,77 @@ def do_login():
     login_success, value = login_using_token()
     if not login_success:
         logger.debug('token login not successful msg: {}'.format(value))
-        if args.silent:
+        if silent:
             logger.debug('exiting because of --silent')
-            sys.exit()
+            return False
         else:
             logger.debug('trying to log in with kano-login')
             run_cmd(dir_path + '/bin/kano-login')
             login_success, value = login_using_token()
             if not login_success:
-                display_msg('Login not possible, error: ' + value, 'error')
-                sys.exit(1)
+                display_msg(
+                    'Login not possible, error: {} error'.format(value),
```

Comment:
Did you mean to add the second `error` after the message? I'm fine with it but the original didn't have it.